scripts/lite/shelley-testnet.sh

```diff
@@ -2,63 +2,78 @@
 
 ROOT="$(realpath "$(dirname "$0")/../..")"
 
-configuration="${ROOT}/configuration"
-
-OS=$(uname -s) DATE=
-case $OS in
-  Darwin )       DATE="gdate";;
-  * )            DATE="date";;
-esac
+configuration="${ROOT}/scripts/lite/configuration"
 
 data_dir="$(mktemp).d"
 mkdir -p "${data_dir}"
 
-# Generate genesis keys
+# Generate shelley genesis
 ARGS=(
-  --genesis-output-dir           "${data_dir}/genesis"
-  --start-time                   "$(${DATE} -d "now + 1 minute" +%s)"
-  --protocol-parameters-file     "${ROOT}/scripts/protocol-params.json"
-  --k                            2160
-  --protocol-magic               459045235
-  --n-poor-addresses             128
-  --n-delegate-addresses         7
-  --total-balance                8000000000000000
-  --avvm-entry-count             128
-  --avvm-entry-balance           10000000000000
-  --delegate-share               0.9
-  --real-pbft
-  --secret-seed                  2718281828
+  --genesis-dir           "${data_dir}/genesis"
+  --gen-genesis-keys      3
+  --gen-utxo-keys         3
+  --supply                9000
+  --mainnet
```

Comment:
Should we be using a testnet magic given the script has the word testnet in it?

Reply:
We can and I've made the change in #1788. If there is nothing majorly wrong with the script, lets get it merged because I need this for #1788  and I'm holding back QA at the moment.